test/bias-2015/test_benign_classifiers.py

```diff
@@ -0,0 +1,453 @@
+"""
+Unit testing for the Bitscopic Interpreting ACMG Standards (BIAS) benign classifiers
+"""
+import os
+import sys
+# NOTE: This should be handled more elegantly for public facing code
+# Some hacky path adjustments so users dont have to fiddle with env variables
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+from src.bias_2015 import benign_classifiers, extract_from_nirvana_json
+
+clinvar_review_status_to_level = {
+    "practice guideline": 5,
+    "reviewed by expert panel": 5,
+    "criteria provided, multiple submitters, no conflicts": 3,
+    "criteria provided, single submitter": 2,
+    "criteria provided, conflicting interpretations": 1,
+    "no assertion criteria provided": 0,
+    "no assertion provided": 0
+}
+
+def test_get_ba1():
+    """
+    Test the get_ba1 function
+    """
+    # Variant with One Thousand Genome AF greater than 5%
+    variant = type('Variant', (object,), {})()
+    variant.oneKg = {'allAf': 0.06}
+    variant.gnomad = None
+    score, ba1 = benign_classifiers.get_ba1(variant)
+    expected_score = 1
+    expected_ba1 = "BA1 (1): One Thousand Genome general population AF=6.00% which is greater than 5%."
+    assert (score, ba1) == (expected_score, expected_ba1)
+
+    # Variant with Gnomad AF greater than 5%
+    variant = type('Variant', (object,), {})()
+    variant.oneKg = None
+    variant.gnomad = {'allAf': 0.07}
+    score, ba1 = benign_classifiers.get_ba1(variant)
```

Comment:
This is a slightly different approach! In general I like to write my unit tests such that each test only tests a single function. There are different approaches to this in the engineering community, and what you've done here (putting multiple function calls in a single test) is another common motif.  

Since our other test modules all have a 1-1 test:function relationship, I think we should do that here to stay consistent.  Can you split these into their own test functions?